paraview/xmls/SignedDistanceField.xml

```diff
@@ -0,0 +1,99 @@
+<ServerManagerConfiguration>
+  <ProxyGroup name="filters">
+    <SourceProxy
+      name="ttkSignedDistanceField"
+      class="ttkSignedDistanceField"
+      label="TTK SignedDistanceField">
+      <Documentation
+        long_help="TTK plugin for the signed distance field computation of surfaces."
+        short_help="TTK plugin for the signed distance field computation of surfaces.">
+
+      </Documentation>
+      
+
+      <InputProperty command="SetInputConnection" name="Input">
+        <ProxyGroupDomain name="groups">
+          <Group name="sources" />
+          <Group name="filters" />
+        </ProxyGroupDomain>
+        <DataTypeDomain name="input_type">
+          <DataType value="vtkDataSet" />
+        </DataTypeDomain>
+        <Documentation>
+          This property specifies the dataset whose data will be probed.
+        </Documentation>
+      </InputProperty>
+      
+      <IntVectorProperty command="SetSamplingDimensions"
+                         default_values="100 100 100"
+                         name="SamplingDimensions"
+                         number_of_elements="3"
+                         panel_visibility="default">
+                         <IntRangeDomain name="range" min="1 1 1"/>
+        <Documentation>
+          Resolution of each axis.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="ExpandBox"
+      command="SetExpandBox"
+      label="Expand Box"
+      number_of_elements="1"
+      default_values="1">
+        <Documentation>
+          This parameter allows to have an exterior border of resolution 1 in the output bounding box compared to the input surface bounding box.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty 
+      name="Backend" 
+      command="SetBackend"
+      number_of_elements="1" 
+      default_values="0"
+      panel_visibility="advanced">
+        <EnumerationDomain name="enum">
+            <Entry value="0" text="Exact"/>
+            <Entry value="1" text="Fast Marching [Sethian, 1996]"/>
+            <Entry value="2" text="Fast Marching Band"/>
+        </EnumerationDomain>
+        <Documentation>
+            Method for the signed distance field computation, exact or approximation (fast marching and fast marching band being a even more approximation).
```

Comment:
"fast marching and fast marching band being a even more approximation"
-> "fast marching and fast marching band, being an even rougher approximation"

Reply:
Done in the next commit!